benchalerts/clients.py

```diff
@@ -399,15 +400,74 @@ def __init__(self, adapter: Optional[HTTPAdapter] = None):
         if login_creds["email"] and login_creds["password"]:
             self.post("/login/", json=login_creds)
 
+    @dataclass
+    class RunComparison:
+        """Track info about a comparison between a contender run and its baseline."""
+
+        contender_info: dict
+        baseline_info: Optional[dict] = None
+        compare_info: Optional[List[dict]] = None
```

Comment:
Could you document what these attributes are at a high level (not schemas, just what it is/where it's from)? I'm assuming `contender_info` and `baseline_info` are data about commits, and `compare_info` is from the compare endpoint, but at a glance all three `*_info`s look kinda the same.

Reply:
Sure!

Reply:
I changed `compare_info` to `compare_results` too. I think that's clearer for what it is.